### PR TITLE
Fix no-content in answer bug

### PR DESCRIPTION
**Pull request by liornoy**
Changed the attributes of RichTextBleachField, from blank=True, null=True
to - default = ''

fix #158

### CODE REVIEW

**Review by avivz450 — APPROVED** [review]

File: home/migrations/0020_alter_answer_content.py [hunk 1]
```diff
@@ -14,6 +14,6 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name="answer",
             name="content",
-            field=home.RichTextBleachField.RichTextBleachField(blank=True, null=True),
+            field=home.RichTextBleachField.RichTextBleachField(default=""),
```
Comment (avivz450) [inline home/migrations/0020_alter_answer_content.py:17]: I think you should also add a test of trying to post an empty answer

**Review by avivz450 — CHANGES_REQUESTED** [review]
